src/runner.rs

```diff
@@ -67,7 +67,12 @@ impl<E: Environment> Runner<E> {
     pub async fn run(&self) -> Result<()> {
         let environments = self.collect_env().await?;
         for env in environments {
-            self.run_env(env).await?;
+            // todo: read env config
+            let db = self.env.start(&env, None).await;
```

Comment:
`env` is confusing here, it's actually `mode`

Reply:
Well... It's true, but I think here `mode` is the name not proper. According to `Environment`'s document the word "env/environment" stands for the start/run mode. I've changed
- the field `env` to `env_controller`
- the parameter `mode` to `env`

What do you think about this?

Reply:
Seems more reasonable than before